### PR TITLE
Allow configuration of contact visualization

### PR DESCRIPTION
**Pull request by SeanCurtis-TRI**
This preserves the historical behavior of visualizing contact forces with fixed-length vectors.

It adds the following:
 - Previously, the fixed-length vectors would be multiplied by a hard-coded scale factor; this is now configurable.
 - There is a minimum magnitude exposed below which forces are not visualized.
 - the "fixed-length" behavior can be toggled (so that vectors linearly scale with force magnitude.

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/11923)

### CODE REVIEW

**Comment by SeanCurtis-TRI** [comment]
Here's what the dialog looks like:

image

**Comment by mposa** [comment]
This is fine, though using a dialog menu without auto-scaling means there's a number of startup clicks necessary to get it in (what I would call) a useful mode. Why not include some form of autoscaling?

**Comment by mposa** [comment]
Not totally sure what's meant by a `drake_visualizer` session vs. historical behavior. Can you rephrase your suggestion?
